src/testup/report/collection.rb

```diff
@@ -5,7 +5,7 @@
 #
 #-------------------------------------------------------------------------------
 
-require 'set'
+require 'testup/report/SortedSet'
```

Comment:
I don't think `SortedSet.rb` should be under the `report` folder. Instead I'd move it up a level to be at `testup/sorted_setrb` (Also adjusting file name to better match existing files)